src/openzaak/components/catalogi/admin/admin_views.py

```diff
@@ -209,3 +209,72 @@ def post(self, request, *args, **kwargs):
             messages.add_message(request, messages.ERROR, exc)
 
         return TemplateResponse(request, self.template_name, context)
+
+
+class ZaaktypePublishView(AdminContextMixin, PermissionRequiredMixin, DetailView):
+    template_name = "admin/catalogi/publish_zaaktype.html"
+    permission_required = "catalogi.change_zaaktype"
+
+    queryset = ZaakType.objects.all()
```

Comment:
Please add `prefetch_related` here. There are loops through elated objects in the template, which could produce a huge pile of DB requests